python/graphstorm/gconstruct/file_io.py

```diff
@@ -132,6 +135,8 @@ def read_data_json(data_file, data_fields):
         for line in json_file.readlines():
             record = json.loads(line)
             data_records.append(record)
+    assert len(data_records) > 0, \
+        f"{data_file} is empty {data_records}."
```

Comment:
if it's empty, why do you need to include data_records in the message?

Reply:
It still contains the column names.